zoloto/cameras/camera.py

```diff
@@ -9,9 +9,13 @@ def __init__(self, camera_id: int, **kwargs):
         self.video_capture = self.get_video_capture(camera_id)
 
     def get_video_capture(self, camera_id):
-        return VideoCapture(camera_id)
+        cap = VideoCapture(camera_id)
+        cap.set(CAP_PROP_BUFFERSIZE)
+        return cap
 
     def capture_frame(self):
+        # Hack: Double capture frames to fill buffer.
+        _, _ = self.video_capture.read()
```

Comment:
is there a need to create dummy assignments here?

```suggestion
        self.video_capture.read()
```